db/migrations/20211215170939-create-account.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+    up: async(queryInterface, Sequelize) => {
+        await queryInterface.createTable('Accounts', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER
+            },
+            name: {
+                allowNull: false,
+                type: Sequelize.STRING
+            },
+            number: {
+                allowNull: false,
```

Comment:
We need to allow nullable value for the account's number because cash accounts have no number